Feature/aws summit 2023 (#792)
ordina-jworks/ordina-jworks.github.io: _posts/2023-06-20-aws-summit-amsterdam-2023.md

{:refdef: style="text-align: center;"}
<img src="{{ '/img/2023-06-20-aws-summit-amsterdam-2023/marielle-lindgren.png' | prepend: site.baseurl }}" alt="Marielle Lindgren" class="image fit" style="margin:0px auto; max-width:100%">
_Marielle Lindgren - carbon neutrality_
{: refdef}

Andy Warfield, the VP & Distinguished Engineer of AWS took over the Keynote and spoke about the newest features and services that AWS has implemented the last year.

{:refdef: style="text-align: center;"}
<img src="{{ '/img/2023-06-20-aws-summit-amsterdam-2023/andy-warfield.png' | prepend: site.baseurl }}" alt="Andy Warfield" class="image fit" style="margin:0px auto; max-width:100%">
_Andy Warfield - the newest features and services_
{: refdef}

Andy also talked about Artificial Intelligence and Machine Learning, which is the newest hype and trend in IT.
Amazon Bedrock, which is presented by Andy, allows you to create generative AI applications by using foundation models.
Their main mission, as always, is that clients should be able to focus only on innovation and their applications, and the purpose of them, and not the surrounding infrastructure.

> With these kinds of topics, users are always afraid of their data and the privacy that surrounds the whole concept.

Reviewer: ```suggestion
With these kinds of topics, privacy is often a concern.
Users want to know how their data is used and what measures are taken to keep it safe.
```

AWS continues to assure that the data of their users remains safe.

> They always design their newest features and services with privacy on the first place, and they will to continue to invest in ways to ensure that they live up to their own standards.

Reviewer: ```suggestion
They always design their newest features and services with privacy as a primary concern, and they will to continue to invest in ways to ensure that they live up to their own standards.
```

In addition to product announcements, the keynote featured customer success stories, highlighting how organizations from various sectors have leveraged AWS to achieve significant business outcomes.

> The speaker spoke of companies that had successfully migrated their infrastructure to the cloud.

Reviewer: ```suggestion
They spoke of companies that had successfully migrated their infrastructure to the cloud.
```

> There were a few companies that were there to share their experience.

Reviewer: ```suggestion
Some of these companies shared their experience on stage.
```

One of the best examples was the Efteling (an amusement park in The Netherlands), which was presented by Jonas Rietbergen.

> They added a functionality to their application where they estimate how occupied the park was at the current moment.

Reviewer: ```suggestion
They added a functionality to their application where they estimate how occupied the park at any point in time.
```

Thanks to these estimations, they were able to predict waiting times and recommend other alternatives in case the current ride was too tied up.
Efteling did this by creating a data lake, integrating it with Sagemaker Model and saving predictions every 30 minutes to Aurora.
These predictions could then be fetched by Lambda and exposed through API Gateway.

{:refdef: style="text-align: center;"}
<img src="{{ '/img/2023-06-20-aws-summit-amsterdam-2023/efteling.png' | prepend: site.baseurl }}" alt="Efteling" class="image fit" style="margin:0px auto; max-width:100%">
_Efteling - predictions explained by Jonas Rietbergen_
{: refdef}

> Other companies were HEMA, PostNL, Epic Games, which were presented by Andy and how these companies used the newest services and features.

Reviewer: Name some services here?

> Takeaway (Just Eat) also gave a small presentation on how they use AWS to improve customer experience in their app.

Reviewer: Again, name 1 or 2 services and how it improved their situation.
